apps/client/package.json

```diff
@@ -6,20 +6,36 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint \"src/**/*.{ts,tsx}\" --quiet",
```

Comment:
When I run `npx nx lint client`, I get the following error. It seems you set the eslint rules but forgot to apply them into the boilerplate code :)
`➜ nx lint client                                   

> nx run client:lint

> client@0.1.0 lint
> eslint "src/**/*.{ts,tsx}" --quiet

/Users/durmusdemirtas/Documents/common-projects/weekend-blog/apps/client/src/app/page.tsx
   49:25  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
   55:24  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
   66:25  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
   72:24  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
   83:25  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
   89:24  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
  100:25  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence
  106:24  error  Curly braces are unnecessary here  react/jsx-curly-brace-presence

✖ 8 problems (8 errors, 0 warnings)
  8 errors and 0 warnings potentially fixable with the `--fix` option.

npm ERR! Lifecycle script `lint` failed with error: 
npm ERR! Error: command failed 
npm ERR!   in workspace: client@0.1.0 
npm ERR!   at location: /Users/durmusdemirtas/Documents/common-projects/weekend-blog/apps/client 

 ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————

 >  NX   Ran target lint for project client (2s)
 
    ✖    1/1 failed
    ✔    0/1 succeeded [0 read from cache]`

Reply:
I left it on purpose, as I assumed that applying linting rules is another task.